### PR TITLE
Saved search "is_bureau" flag

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds a new column to the SavedSearch model to indicate whether the saved search is bureau/AO-related. Default to false. Currently no implementation on front-end.

### CODE REVIEW

File: talentmap_api/user_profile/views/searches.py [hunk 1]
```diff
@@ -41,7 +42,7 @@ class SavedSearchView(FieldLimitableSerializerMixin,
 
     serializer_class = SavedSearchSerializer
     permission_classes = (IsAuthenticated,)
-    filter_backends = [CaseInsensitiveOrderingFilter]
```
Comment (elizabeth-jimenez) [inline talentmap_api/user_profile/views/searches.py:44]: I think we can remove the import for `CaseInsensitiveOrderingFilter `

Comment (mjoyce91) [reply talentmap_api/user_profile/views/searches.py:44]: Good catch - I'll test it out and make sure sorting still works

Comment (mjoyce91) [reply talentmap_api/user_profile/views/searches.py:44]: Oh wait nvm I see what you mean

Comment (elizabeth-jimenez) [reply talentmap_api/user_profile/views/searches.py:44]: hehe I was like 👀  i must be completely lost

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍥 🥮 👍

**Pull request merged**